core/feature_flag_list.py

```diff
@@ -50,6 +50,7 @@ class FeatureNames(enum.Enum):
     LEARNER_GROUPS_ARE_ENABLED = 'learner_groups_are_enabled'
     NEW_LESSON_PLAYER = 'new_lesson_player'
     CD_ALLOW_UNDOING_TRANSLATION_REVIEW = 'cd_allow_undoing_translation_review'
+    UPDATE_EXP_VOICE_ARTIST_LINK = 'update_exp_voice_artist_link'
```

Comment:
Given the description, could we maybe perhaps call this ``AUTO_UPDATE_EXP_VOICE_ARTIST_LINK``, instead? Prefer being more verbose for flags since we'll be adding lots of them (and the strings we chose stick around essentially forever).

Reply:
Done